### PR TITLE
feat(WalletFirst): Rearrange left nav bar and open wallet section by default

### LINKED ISSUE #16778: [Wallet first] Reorder left side bar and move the wallet as the first option
Acceptance criteria:

- Wallet is loaded after onboarding
- Wallet is the first item in the left side bar
- The active session is persisted after restart

### PR DESCRIPTION
**Pull request by alexjba**
### What does the PR do

Closes #16778

1. Wallet section is the default section
2. Wallet section is the first section in the left nav bar

Acceptance criteria:

Wallet is loaded after onboarding
Wallet is the first item in the left side bar
The active session is persisted after restart

### Affected areas

Left nav bar
The default active section
Active section persistency

### Architecture compliance

- [x] I am familiar with the application architecture and agreed good practices.
My PR is consistent with this document: [Status Desktop Architecture Guide](https://github.com/status-im/status-desktop/blob/master/CONTRIBUTING.md)

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://github.com/user-attachments/assets/2cd66578-1124-4b00-aa09-46277e7ca446

### User stories:
```
Given the user has logged in
When the app starts
Then the wallet button is the first one in the left navigation bar
```
```
Given the user has signed up for a new account
When the onboarding process ends
Then the wallet section is displayed
```
```
Given the user has selected [chat,wallet,community,communityPortal] section
And quits the app
When the user logs in again
Then the last selected section is displayed
```
```
Given the user has selected settings section
And quits the app
When the user logs in again
The the wallet section is displayed
```

### CODE REVIEW

**Review by alaibe — APPROVED** [review]

**Comment by alexjba** [comment]
@anastasiyaig Looks like we need to update the e2e as well

**Comment by anastasiyaig** [comment]
ya i will do it soon @alexjba

File: ui/imports/utils/Constants.qml [hunk 1]
```diff
@@ -320,9 +320,9 @@ QtObject {
     readonly property int chatSectionLeftColumnWidth: 304
 
     readonly property QtObject appSection: QtObject {
-        readonly property int chat: 0
-        readonly property int community: 1
-        readonly property int wallet: 2
+        readonly property int chat: 1
+        readonly property int community: 2
+        readonly property int wallet: 0
```
[the remark this answers is not in the document]
Comment (alexjba) [reply ui/imports/utils/Constants.qml:325]: I didn't find a reason to do it. The `appViewStackIndex` is mapped with `appSection` in AppMain and it's configuring the index in the StackView.
I've also seen that `appViewStackIndex` didn't match `appSection` before adding these changes.

**Review by caybro — APPROVED** [review]

**Comment by anastasiyaig** [comment]
i tested that and it works matching the acceptance criteria defined
deep links are also working, tried community link, link to profile, link to send in chat
i also finally fixed autotests (commit in this PR) 
this is good to be merged

**Review by anastasiyaig — APPROVED** [review]

**Review by micieslak — APPROVED** [review]
Good job!

Btw, it would be nice to get rid of the concept of "section" from the NIM side and handle the app layout exclusively on the UI side. But I'm not sure how big effort it is.

File: ui/app/mainui/AppMain.qml [hunk 2]
```diff
@@ -669,20 +669,21 @@ Item {
...
+            topSectionModel: SortFilterProxyModel {
                 sourceModel: appMain.rootStore.mainModuleInst.sectionsModel
                 filters: [
-                    ValueFilter {
+                    RangeFilter {
                         roleName: "sectionType"
-                        value: Constants.appSection.chat
+                        minimumValue: Constants.appSection.wallet
+                        maximumValue: Constants.appSection.chat
                     },
```
[the remark this answers is not in the document]
Comment (alexjba) [reply ui/app/mainui/AppMain.qml:679]: yes, good point! Using min, max values is quite fragile.

[the remark this answers is not in the document]
Comment (alexjba) [reply ui/app/mainui/AppMain.qml]: updated

**Pull request merged**